packages/light-apps/src/TopBar/ChooseProvider/ChooseProvider.tsx

```diff
@@ -0,0 +1,55 @@
+import { Dropdown, DropdownProps } from '@substrate/ui-components';
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+
+import { ProviderContext } from '../../ContextGate/context';
+
+const options = [
+  {
+    key: 'PostMessageProvider|kusamaCc3',
+    text: 'Kusama CC3 (from extension Light Client)',
+    value: JSON.stringify({
+      payload: 'kusamaCc3',
+      type: 'PostMessageProvider',
+    }),
+  },
+  {
+    key: 'PostMessageProvider|westend',
+    text: 'Westend (from extension Light Client)',
+    value: JSON.stringify({
+      payload: 'westend',
+      type: 'PostMessageProvider',
+    }),
+  },
+  {
+    key: 'WsProvider|kusamaCc3',
+    text: 'Kusama CC3 (from centralized remote node)',
+    value: JSON.stringify({
+      payload: 'wss://kusama-rpc.polkadot.io',
+      type: 'WsProvider',
+    }),
+  },
+];
```

Comment:
Similar question to above: will this eventually work with arbitrary chains?

Reply:
Yeah, for the next milestone (btw i created a milestone on github https://github.com/paritytech/substrate-light-ui/milestone/3), we could add the functionality of user uploading a chain spec